horace_core/sqw/coord_transform/@aProjection/private/check_angdeg_.m

```diff
@@ -0,0 +1,21 @@
+function angdeg = check_angdeg_(val)
+% verify if input lattice angles belong to acceptable range
+%
+if numel(val) == 1
+    val = abs(val);
+    angdeg = [val,val,val];
+elseif size(val,2)==3 && size(val,1)==1
+    angdeg = abs(val);
+elseif size(val,1)==3 && size(val,2)==1
+    angdeg = abs(val');
+else
```

Comment:
Isn't this just a repeat of `check_alatt_`? Would it not be more useful to add a `to_three_vector` with this functionality? And catch the throw to add relevant info?

Reply:
it is some repetition, but the whole range of checks is different. not sure, if it worth splitting relatively simple check into two parts -- some loss of the code readability against some decrease in code repetition -- do not feel it worth it.